quiz/src/main/java/com/epam/spring/homework3/quiz/controller/QuizController.java

```diff
@@ -0,0 +1,77 @@
+package com.epam.spring.homework3.quiz.controller;
+
+import com.epam.spring.homework3.quiz.controller.assembler.QuizAssembler;
+import com.epam.spring.homework3.quiz.controller.assembler.model.QuizModel;
+import com.epam.spring.homework3.quiz.controller.dto.QuizDto;
+import com.epam.spring.homework3.quiz.controller.dto.group.OnCreate;
+import com.epam.spring.homework3.quiz.controller.dto.group.OnUpdate;
+import com.epam.spring.homework3.quiz.controller.mapper.QuizMapper;
+import com.epam.spring.homework3.quiz.service.QuizService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequestMapping("/quiz")
+@RequiredArgsConstructor
+@Api(tags = "QuizController description for SWAGGER documentation")
+public class QuizController {
```

Comment:
same about url naming:

`getQuizByTitle` -> GET `/quiz/{title}`
`updateQuizByTitle` -> `PUT /quiz/{title}`
`createQuiz` -> good, no changes needed
`deleteQuizByTitle` -> DELETE `/quiz/{title}`

This one is interesting (in order to find quizes created by user - here's how it can be done)
In order to achive this:
- you would need to remove `@RequestMapping("/quiz")` on top of QuizController
- keep this method inside the QuizController `getAllQuizesByCreatorId`

`getAllQuizesByCreatorId` -> `GET /users/{id}/quiz`